packages/compass-collection/src/components/collection/collection.tsx

```diff
@@ -33,6 +34,52 @@ const collectionModalContainerStyles = css({
   zIndex: 100,
 });
 
+const getInsightsForPipeline = (pipeline: Document[], isAtlas: boolean) => {
+  const insights: Record<string, Signal> = {};
+  for (const stage of pipeline) {
+    if ('$lookup' in stage) {
+      const signal = {
+        id: 'lookup-usage-in-view',
+        title: 'This view uses $lookup',
+        description:
+          '$lookup operations can be resource intensive because they perform operations on two collections instead of one. Consider embedding documents or arrays to increase read performance.',
```

Comment:
I mentioned this a bit in a comment on the tech design, I think we want to be sure we're not leading folks down the wrong path and causing schema mishaps. Would this text make it more non-suggestive?
```suggestion
          '$lookup operations can be resource intensive because they perform operations on two collections instead of one. In certain situations, consider embedding documents or arrays to enhance read performance.',
```

Reply:
Thank you Rhys and sorry for we actually resolved the thread after some inactivity. The suggestion is definitely more neutral. I will check in once with Julia as well for the copy. Was there any other feedback?

Reply:
There was no activity on the thread so we resolved it. Let's move this stage out of the PR if there is still stuff to figure out

Reply:
That sounds good. I will get on it.